src/main/java/com/fasterxml/jackson/dataformat/xml/ser/XmlSerializerProvider.java

```diff
@@ -56,6 +55,11 @@ public XmlSerializerProvider(XmlSerializerProvider src,
      */
 
     @Override
+    public DefaultSerializerProvider copy() {
+        return this;
```

Comment:
I can add a comment if needed, but I'm doing this because `XmlSerializerProvider` is effectively immutable in that it's only member variable is also immutable, and so I can avoid having to try to figure out whether I should construct it with just an instance of `XmlRootNameLookup` or with an instance of itself and `SerializationConfig` and `SerializerFactory`